go/libkb/constants.go

```diff
@@ -71,7 +71,15 @@ const (
 // Right now reddit is the only site that seems to have any requirements for
 // our User-Agent string. (See https://github.com/reddit/reddit/wiki/API.)If
 // something else comes up, we'll want to make this more configurable.
-var UserAgent = runtime.GOOS + ":" + "Keybase CLI (" + runtime.Version() + "):" + Version
+func UserAgent () string {
+	// Twitter and Facebook serve up mobile versions of their sites if
+	// our useragent contains "android", breaking PVL; pretend to be Linux.
+	var OS = runtime.GOOS
+	if runtime.GOOS == "android" {
+		OS = "linux"
+	}
+	return OS + ":" + "Keybase CLI (" + runtime.Version() + "):" + Version
```

Comment:
Reddit asks us to include the platform (at the link mentioned there, https://github.com/reddit/reddit/wiki/API), but it seems like respecting platform differences can only hurt us here. Should we get rid of that field? (Possibly breaking something on their end if they really do try to parse this?) Or maybe tell a white lie and always say it's `linux`? @chromakode do you know what reddit would prefer us to do here?